pkgs/by-name/on/oneanime/package.nix

```diff
@@ -0,0 +1,120 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter324,
+  autoPatchelfHook,
+  wrapGAppsHook3,
+  makeDesktopItem,
+  pkg-config,
+  copyDesktopItems,
+  alsa-lib,
+  libepoxy,
+  libpulseaudio,
+  libdrm,
+  mesa,
+  xdg-user-dirs,
+  libva,
+  libva1,
+  libvdpau,
+  buildGoModule,
+}:
+let
+  libopencc = buildGoModule rec {
+    pname = "libopencc";
+    version = "1.0.0";
+
+    src = fetchFromGitHub {
+      owner = "Predidit";
+      repo = "open_chinese_convert_bridge";
+      rev = "refs/tags/${version}";
+      hash = "sha256-kC5+rIBOcwn9POvQlKEzuYKKcbhuqVs+pFd4JSFgINQ=";
+    };
+
+    vendorHash = "sha256-ADODygC9VRCdeuxnkK4396yBny/ClRUdG3zAujPzpOM=";
+
+    buildPhase = ''
+      runHook preBuild
+
+      go build -buildmode=c-shared -o ./libopencc.so
+
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+
+      install -Dm0755 ./libopencc.so $out/lib/libopencc.so
+
+      runHook postInstall
+    '';
+
+    meta = {
+      homepage = "https://github.com/Predidit/open_chinese_convert_bridge";
+      license = with lib.licenses; [ gpl3Plus ];
+      sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
```

Comment:
```suggestion
```

opencc is a go project, and so is its wrapper. I mean, I don't know why you would set the platform here, since we don't have upstream binaries?